stubs/redis/redis/asyncio/client.pyi

```diff
@@ -22,12 +22,12 @@ ResponseCallbackT: TypeAlias = ResponseCallbackProtocol | AsyncResponseCallbackP
 
 class Redis(AbstractRedis, RedisModuleCommands, AsyncCoreCommands[_StrType], AsyncSentinelCommands, Generic[_StrType]):
     response_callbacks: MutableMapping[str | bytes, ResponseCallbackT]
-    @classmethod
-    def from_url(cls, url: str, **kwargs) -> Redis[Any]: ...
-    auto_close_connection_pool: Any
+    auto_close_connection_pool: bool
     connection_pool: Any
     single_connection_client: Any
     connection: Any
+    @classmethod
+    def from_url(cls, url: str, **kwargs) -> Redis[Any]: ...
```

Comment:
Why move this method down a few lines? It doesn't look like the signature's changed.

Reply:
So attributes and methods are not mixed. It was hard to read.

Reply:
Okay, fair enough :)